### ISSUE TITLE
How are images such as depth maps and segmentation maps in the dataset created?

### ISSUE THREAD

**Issue by cpy0029** [VIPL-GENUN/Jodi#4]
If I want to fine-tune Jodi on other datasets, how should I generate depth maps, sketches, and segmentation maps corresponding to my dataset? Thanks !!!!!

**Comment by xyfJASON** [comment]
We use pretrained annotators to generate these labels. Specifically:
- Depth: [Depth Anything V2](https://github.com/DepthAnything/Depth-Anything-V2) and [Lotus](https://github.com/EnVision-Research/Lotus)
- Normal: [Lotus](https://github.com/EnVision-Research/Lotus)
- Line Art: [Informative Drawings](https://github.com/carolineec/informative-drawings)
- Edge: [PiDiNet](https://github.com/hellozhuo/pidinet)
- Albedo: [RGB2X](https://github.com/zheng95z/rgbx)
- Segmentation: [OneFormer](https://github.com/SHI-Labs/OneFormer)
- Human pose: [OpenPose](https://github.com/CMU-Perceptual-Computing-Lab/openpose)
